src/test/net/SSLHTTPDownloader_test.cpp

```diff
@@ -17,11 +17,10 @@
 */
 //==============================================================================
 
-#include <ripple/net/SSLHTTPDownloader.h>
+#include <ripple/net/DatabaseDownloader.h>
```

Comment:
This file is now a `DatabaseDownloader` test, not an `SSLHTTPDownloader` test.  Please fix the name of the file, all comments, and the `BEAST_DEFINE_TESTSUITE` macro at the bottom of the file.

Reply:
Done